pkg/trace/writer/stats_test.go

```diff
@@ -119,6 +119,7 @@ func TestStatsWriter(t *testing.T) {
 				Sequence:         34,
 				AgentAggregation: "aggregation",
 				Service:          "service",
+				ContainerID:      "container-id",
```

Comment:
If time allows, or perhaps in a follow-up PR, it may make sense to have a regression test. Or does this change cover it?